x-pack/plugins/reporting/common/types.ts

```diff
@@ -98,12 +98,20 @@ export interface ReportDocument extends ReportDocumentHead {
 }
 
 export interface BaseParams {
-  browserTimezone?: string; // browserTimezone is optional: it is not in old POST URLs that were generated prior to being added to this interface
   layout?: LayoutParams;
   objectType: string;
   title: string;
 }
 
+/*
+ * Job Params are defined by the application, and on the client side, are decorated by the Reporting service.
+ * These are required for handling additional formatting or state migrations
+ */
+export type DecoratedBaseParams = BaseParams & {
```

Comment:
whats the reason for adding `DecoratedBaseParams` instead of just adding browserTimezone and version to `BaseParams` ?

Reply:
This allowed the `getJobParams` functions of the various apps to be correctly typed in saying they return `BaseParams`.

Reporting has still has trouble spots in the code where apps can not really import the correct type that they want to, because those types are restricted to the reporting code, on the server side. 

I've undone the addition of `DecoratedBaseParams`: 300ba45f6c4